action.py

```diff
@@ -24,6 +24,7 @@ def __init__(
         theme_context_light: Dict[str, str],
         theme_context_dark: Dict[str, str],
         show_duration: bool,
+        output_type: Optional[str] = "markdown",
```

Comment:
Rather than putting a default here, you can add a line to `test_action.py` (around line 19) passing "markdown" to output_type.

It would also be great if you could add a test there (eg. `test_parse_videos_html`) that changes the _output_type to "html" and tests parsing the videos as html.